### ISSUE TITLE
Bill summary: truncate and style

### ISSUE THREAD

**Issue by aih** [aih/FlatGov#236]
The bill summary now has the correct data. But it is not styled and may be too long (see below). We should truncate it by default (200 characters?) and provide a toggle to allow the user to show the whole summary. We should also style it, to the degree possible, to differentiate headings from text.

![image](https://user-images.githubusercontent.com/217356/111839170-9b403080-88b7-11eb-940c-8155b0ebbe92.png)

**Comment by aih** [comment]
Use `pre` element to wrap the summary and css to manage the overflow. See, e.g. 

https://stackoverflow.com/a/43851641/628748

```javascript
.pre-wrapper {
    border: 1px solid #d2d2d2;
    margin:20px;
    overflow-x: scroll;
    position: relative;
  }
  .pre-wrapper > div {
    display: table;
    width: 100%;
  }
  .pre-wrapper pre {
    background:transparent;
    border:0;
    border-bottom:1px solid #eee;
    border-radius:0;
    display: block;
    height:28px;
    line-height:28px;
    margin:0;
    min-width:100%;
    overflow: visible;
    padding:0;
    padding-left:10px;
    padding-right:10px;
    position: relative;
    white-space: pre;
    word-wrap:normal;
  }
  .pre-wrapper pre.added {
    background-color:#ddffdd;
  }
  .pre-wrapper pre.removed {
    background-color:#fee8e9;
  }
```

**Comment by aih** [comment]
Fixed in #237
